docs/parent/pom.xml

```diff
@@ -41,6 +41,8 @@
         <html.rel.file.prefix></html.rel.file.prefix>
         <mq.docs.url>https://eclipse-ee4j.github.io/openmq/guides/</mq.docs.url>
         <bookDirectory>${project.build.directory}/book</bookDirectory>
+        <productMajorVersion>${parsedVersion.majorVersion}</productMajorVersion>
```

Comment:
This won't work - the `parsedVersion.majorVersion` is resolved later and is not available at this time. You need to define this property after the `parse-version` goal is executed, e.g. with the antrun plugin:

```
<plugin>
  <groupId>org.apache.maven.plugins</groupId>
  <artifactId>maven-antrun-plugin</artifactId>
  <version>3.0.0</version>
  <executions>
    <execution>
      <phase>initialize</phase>
      <goals>
        <goal>run</goal>
      </goals>
      <configuration>
        <target>
          <property name="productMajorVersion" value="${parsedVersion.majorVersion}"/>
        </target>
      </configuration>
    </execution>
  </executions>
</plugin>
```